File: app/src/main/java/com/vmb/financialcalculator/MainActivity.java
package com.vmb.financialcalculator;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void onClickStartDeposit(View view){
        Intent intent = new Intent(this, DepositActivity.class);
        startActivity(intent);
    }

    public void onClickStartCredit(View view){
        Intent intent = new Intent(this, CreditActivity.class);
        startActivity(intent);
    }
}

File: app/src/main/java/com/vmb/financialcalculator/MainActivity.kt
package com.vmb.financialcalculator

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun onClickStartDeposit(view: View?) {
        val intent = Intent(this, DepositActivity::class.java)
        startActivity(intent)
    }

    fun onClickStartCredit(view: View?) {
        val intent = Intent(this, CreditActivity::class.java)
        startActivity(intent)
    }
}